src/dragonball/src/metric.rs

```diff
@@ -77,6 +78,8 @@ pub struct DragonballMetrics {
     pub rtc: Arc<RTCDeviceMetrics>,
     /// Metrics related to serial device.
     pub serial: HashMap<String, Arc<SerialDeviceMetrics>>,
+    /// Metrics related to a MMIO transport.
+    pub mmio: HashMap<String, Arc<MMIODeviceMetrics>>,
```

Comment:
It's better to add mmio transport layer's metric to each device's metric

Reply:
I agree with you. I will consider it later.